### DISCUSSION TITLE
2023년 05월 16일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#54] [General]
# 2023년 05월 16일

⚖️ 공부한 것, 일기를 적는 개인 위키입니다 인덱스 페이지: [ wiki-index ] 이번 달의 TIL: [ 2023-05|2023년 5월의 TIL✨ ] 현재 자주 쓰는 글들: [ wooteco|우아한테크코스 ]

https://vero.wiki/2023-05-16/

**Comment by greeng00se** [comment]
마늘종이 뭘까?

**Comment by be-student** [comment]
대체 누누 등장을 얼마나 많이 해야되죠?ㅋㅋㅋㅋ

**Reply by be-student** [reply]
다음 글 주세요!

**Comment by kevstevie** [comment]
오늘 5월 21일이에요

**Reply by ezzanzzan** [reply]
곧 22일이에요

**Reply by drunkenhw** [reply]
https://drunkenhw.github.io/java/Domain-vs-Entity/
이거라도 보세요
